website/content/docs/concepts/oidc-provider.mdx

```diff
@@ -0,0 +1,152 @@
+---
+layout: docs
+page_title: OIDC Provider
+description: >-
+  Describes how Vault can be an OIDC identity provider.
+---
+
+# OIDC Provider
+
+~> **Note**: The Vault OIDC Provider feature currently only supports the [authorization code flow](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth).
+
+This document describes how Vault can be an **OpenID Connect(OIDC) identity provider** by enabling applications to leverage Vault as a source of identity using the OIDC protocol.
+
+This feature allows clients speaking the OIDC protocol to take advantage of Vault's various authentication methods and source of identity. Clients can configure their authentication logic to talk to Vault. Once enabled, Vault will act as the bridge to identity providers via its existing authentication methods. Clients will also obtain identity information for their end-users by leveraging custom templating of Vault identity information.
+
+Vault as an OIDC provider allows mutual Vault and Boundary customers to leverage Vault's identity system to delegate authentication and authorization to Vault.  Vault, therefore, acts as an identity provider for Boundary. Other HashiCorp products such as Consul can also leverage Vault's identity system and provide delegated authentication and authorization to its users. Having Vault as an OIDC provider allows a single sign-on experience to their end-users for organizations that want to leverage Vault as an identity provider.
+
+## Configuration Options
+
+The next few sections of the document provide implementation details for each resource that permits Vault configuration as an OIDC identity provider.
+
+### Providers
+
+A Vault namespace may contain several provider resources. Each configured provider will publish the APIs listed within the OIDC flow. The APIs will be served via backend path-based routing on Vault's listen [address](/docs/configuration/listener/tcp#address).
+
+A provider must have the following configuration parameters:
+
+* **Issuer URL**: used in the `iss` claim of ID tokens
+* **Allowed client IDs**: limits which clients can access the provider
+* **Scopes supported**: limits what identity information is available as claims.
```

Comment:
Nit: consistency with period at end of list items

Reply:
fixed.